main_routers/config_router.py

```
@@ -0,0 +1,354 @@
+
+from .shared_state import get_config_manager, get_steamworks, get_session_manager, get_initialize_character_data
+from utils.preferences import load_user_preferences, update_model_preferences, validate_model_preferences, move_model_to_top
+from utils.frontend_utils import find_models, find_model_config_file
+
+router = APIRouter(prefix="/api", tags=["config"])
+logger = logging.getLogger("Main")
+
+
+@router.get("/config/page_config")
+async def get_page_config(lanlan_name: str = ""):
+    """获取页面配置（lanlan_name 和 model_path）"""
```

Ruff (0.14.8)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)